### PR TITLE
Update brain_build.md

### PR DESCRIPTION
**Pull request by Daibread**
1. Move PCB item to be right before jumper cables ( labled "consumable-A")
2. identify jumper cables as such
3. explain that rasp3 is alternative
4. explain that consumable-b is for mounting on touch-screen, in case of non-PCB build
5. move brain build instructions to the same page as BOM

### CODE REVIEW

File: 4_Manufacturing_Design/Alpha_Build_Instructions/brain_build.md [hunk 1]
```diff
@@ -61,4 +60,45 @@ For how to assemble the brains, check this wiki [here](brain_build_instructions.
 [2mono]:https://www.monoprice.com/product?p_id=3896
 [7mono]:https://www.monoprice.com/product?p_id=31201
 [7-alt-mono]:https://www.monoprice.com/product?p_id=13924
-[12rw]:https://github.com/respiraworks/pcbreathe
```
Comment (martukas) [inline 4_Manufacturing_Design/Alpha_Build_Instructions/brain_build.md:64]: Why did you remove this? This breaks the link on the line you moved up to line 31 :(

Comment (Daibread) [reply 4_Manufacturing_Design/Alpha_Build_Instructions/brain_build.md:64]: Didn't knowingly remove it, however have replaced it and commited the change
